config.schema.json

```diff
@@ -25,6 +25,12 @@
                 "title": "Absolute Max Occupation Time (auto-shutoff)",
                 "type": "number"
             },
+            "occupancyLogging": {
+                "default": true,
+                "description": "Write logs when a sensor becomes occupied or unoccupied.",
```

Comment:
For consistency of language, let's replace "a sensor" with "the occupancy sensor"

Reply:
Sounds good! I have made this adjustment in d05c4e5.